cmake/inputs/patches/ep_bzip2/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+#
+# CMakeLists.txt
+#
+#
+# The MIT License
+#
+# Copyright (c) 2017 TileDB, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+project(bzip2)
+cmake_minimum_required(VERSION 2.8)
+
+set(SOURCES "blocksort.c" "huffman.c" "crctable.c" "randtable.c"
+"compress.c" "decompress.c" "bzlib.c")
+file(GLOB HEADERS "bzlib.h")
+
+# Set build flags
+if (WIN32)
```

Comment:
This needs to be `if (MSVC)` because we also build w/ mingw on Windows (not sure if bzip2 builds from source for the R packaging, but better to be safe).

Reply:
@ihnorton
That (CMakeLists.txt) is the same as it has been with prior 1.0.6 bzip2 version for (3?) years.
It only shows up in the diff because Seth asked that I place it in a different place in the repository requiring an 'add' in that location.
While I can make that change, I have to wonder if it would instead cause something to break in those other environments, ?

The substantive changes were
A) pulling original 1.0.8 source archive from primary site (instead of our re-manufactured archive with one additional file)
B) merging our CMakeLists.txt into it (rather than having had a re-manufactured archive containing it that we pull and used)

Reply:
Alright, thanks for the explanation. Fine to keep it conservative for now.